Clarification on parcels (#207)
CanopyTax/single-spa.js.org: website/versioned_docs/version-5.x/microfrontends-concept.md

2. [single-spa parcels](/docs/parcels-overview): Microfrontends that render components without controlling routes.
3. [utility modules](/docs/recommended-setup#utility-modules-styleguide-api-etc): Microfrontends that export shared javascript logic, without rendering components.

> A web app may include one or more types of microfrontends. See the [differences broken down](/docs/module-types), and our recommendations on [choosing between microfrontend types](/docs/recommended-setup#applications-versus-parcels-versus-utility-modules).

Reviewer: nit: "See the differences broken down" is a bit harder for non-native english speakers and for our translators:

```suggestion
A web app may include one or more types of microfrontends. See [an in-depth comparison](/docs/module-types), and our recommendations for [choosing between microfrontend types](/docs/recommended-setup#applications-versus-parcels-versus-utility-modules).
```
